pkg/api/annotation_key_constants.go

```diff
@@ -19,6 +19,10 @@ limitations under the License.
 package api
 
 const (
+	// ImagePolicyFailedOpenKey is added to pods created by failing open when the image policy
+	// webhook backend fails.
+	ImagePolicyFailedOpenKey string = "alpha.image-policy.k8s.io/failed-open"
```

Comment:
I based the name off the "filterAnnotations" function in this file. It checks this "*.image-policy.k8s.io/*"

I'm OK with "image-policy.*.k8s.io/*", but the naming inconsistency sounds like a good thing to fix from alpha > beta.
.